datasets/bg/jud_declarations/bg_jud_declarations.yml

```
@@ -0,0 +1,69 @@
+title: Bulgaria Judicial Declarations
+entry_point: crawler.py
+prefix: jud-declarations
+coverage:
+  frequency: weekly
+  start: 2025-01-17
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  List of judges, prosecutors, and investigators in Bulgaria who are required to submit asset 
+  declarations.
+description: |
+  This dataset provides a list of judges, prosecutors, and investigators in Bulgaria. Published 
+  asset declarations under Article 175a, are linked to each individual's profile.
```

Maybe we can flesh out a bit like this? But can you identify the act?

```suggestion
description: |
  This dataset provides a list of judges, prosecutors, and investigators in Bulgaria with links
  to their asset declarations.
  
  Article 175a of act ??? requires senior members of the judiciary to submit asset
  declarations annually. This provides us with a list of politically-exposed persons
  within the judiciary of Bulgaria.
```